_scripts/_run_jekyll.sh

```diff
@@ -15,12 +15,17 @@ mv $WRKDIR/documentation-generator/new_references.md $WRKDIR/documentation-gener
 mkdir $WRKDIR/documentation-generator/pages
 cp -r $WRKDIR/documentation/* $WRKDIR/documentation-generator/pages
 cd $WRKDIR/documentation-generator
+# rvm commands are insane scripts which pollut output
+# so instead of set -x we just echo each command ourselves
+set +x
+echo "+ /home/jenkins/.rvm/scripts/rvm"
 source /home/jenkins/.rvm/scripts/rvm
 echo "Latest jekyll run :$BUILD_ID" > $WRKDIR/output.log
 echo "Based on latest git commit :$GIT_COMMIT" >> $WRKDIR/output.log
 echo "*********************************************************" >> $WRKDIR/output.log
 echo "*                  CONSOLE OUTPUT                       *" >> $WRKDIR/output.log
 echo "*********************************************************" >> $WRKDIR/output.log
+set -x
 jekyll
 if [ "$?" -gt "0" ]; then
```

Comment:
if it fails, perhaps we should cat the output.log

Reply:
I'm not really sure, but it looks like `jekyll` outputs directly to stdout, while $WRKDIR/output.log contains only some fancy output like "Latest jekyll run" and "Based on latest git commit"

Note that `set -x` _enables_ outputting of runned commands to console.